BAZProjectC1/Home/ViewModel/MovieAPI.swift

```diff
@@ -9,25 +9,19 @@ import UIKit
 final class MovieAPI {
     private let apiKey: String = "f6cd5c1a9e6c6b965fdcab0fa6ddd38a"
     
-  public func getMovies(completion: @escaping ([Movie])->()){
-        guard let url = URL(string: "https://api.themoviedb.org/3/trending/movie/day?api_key=\(apiKey)")
+    /// Método utilizado para consumir la lista de películas correspondientes acordes al filtro seleccionado.
+    ///  - returns: Si el consumo es exitoso, un array con las películas obtenidas; en caso contrario un array vacío
```

Comment:
Realmente este método no tiene valores de retorno. Tiene como parámetros un string y un clousure, eso es lo que hay que mencionar. 

https://developer.apple.com/documentation/xcode/writing-symbol-documentation-in-your-source-files

Esta guia te pede ayudar a documentar mejor tus métodos.